web_infrastructure/jboss_cli.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016, Roland Wolters <rwolters@redhat.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+DOCUMENTATION = '''
+---
+module: jboss_cli
+version_added: "2.1"
+author: "Roland Wolters (@liquidat)"
+short_description: Manage JBoss via jboss_cli
+requirements:
+    - jboss_cli
+description:
+    - This module provides JBoss management via jboss_cli for JBoss application servers
+options:
+    user:
+        description:
+            - the name of the management user
+        required: false
+    password:
+        description:
+            - the password to assign to the username
+        required: false
+    host:
+        description:
+            - the host running JBoss
+        required: false
+        default: '127.0.0.1'
+    port:
+        description:
+            - the port of the associated host running JBoss
+        required: false
+        default: '9999'
+    jbosshome:
+        description:
+            - The JBoss path environment variable
+        required: false
+        default: '/usr/share/jbossas/'
+    command:
+        description:
+            - The command to manage the JBoss server
+        required: true
+'''
+EXAMPLES = '''
+Example playbook entries using the jboss_cli module to manage JBoss.
+
+    tasks:
+
+    - name: set max-threads to a given value
+      jboss_cli: command="/subsystem=ejb3/thread-pool=default:write-attribute(name=max-threads,value=3000)"
```

Comment:
I took the examples from the official documentation at https://docs.jboss.org/author/display/WFLY8/CLI+Recipes - can you point me to a source about the `key: value` principle?